sdk/rust/src/http.rs

```diff
@@ -99,6 +384,23 @@ impl std::fmt::Display for Method {
 }
 
 impl IncomingRequest {
+    /// The incoming request Uri
+    pub fn uri(&self) -> String {
```

Comment:
Can the user get this as a Url object or would they need to parse the string?

Reply:
They would need to parse. The thought by behind it is to not make the `http` crate part of the public API for these types.